tools_iuc.yaml

```diff
@@ -3364,62 +3364,65 @@ tools:
     owner: iuc
     tool_panel_section_label: RAD-seq
 
-  - name: humann2
+  - name: humann
     owner: iuc
     tool_panel_section_label: Metagenomic Analysis
 
-  - name: humann2_associate
+  - name: humann_associate
```

Comment:
I would leave the old tools, we don't want to remove them left? We want to hide them?

Side note: removing tools from the yaml does not remove it from Galaxy. This is not implemented.

Reply:
ok. I will revert that then